kernel/src/fs/epoll/entry.rs

```diff
@@ -31,6 +34,15 @@ pub(super) struct Entry {
     observer: Arc<Observer>,
 }
 
+impl Display for Entry {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        let inner = self.inner.lock();
+        write!(f, "tfd:\t{} ", self.key.fd)?;
+        write!(f, "events:\t{:x} ", inner.event.events.bits())?;
+        write!(f, "data: {:x}", inner.event.user_data)
```

Comment:
This is not correct. The Linux implementation is `tfd: %8d events: %8x data: %16llx`.